hw_2/templates/welcome_task_9.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+{% block content %}
+
+<p>Welcome, {{username}}!</p>
+<form action="{{url_for('delete_cookies')}}">
```

Comment:
И тут. но понадобиться атрибут `method `
```suggestion
<form method="post">
```